__tests__/app.test.js

```diff
@@ -85,10 +86,46 @@ describe("GET /api/articles", () => {
         );
       });
   });
-  // it("should return an array of objects", () => {
-  //   return request(app)
-  //     .get("/api/articles")
-  //     .expect(200)
-  //     .then((res) => {
-  //     }
+  it("should return an array of objects", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then((res) => {
+        expect(Array.isArray(res.body.result)).toBe(true);
+        res.body.result.forEach((article) =>
+          expect(typeof article).toBe("object")
+        );
+      });
+  });
+  it("should return articles in desc date order", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then((res) => {
+        expect(res.body.result).toBeSorted({ descending: true });
+      });
+  });
+  it("should have all properties correct", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then((res) => {
+        res.body.result.forEach(
+          ((article) => expect(article).toHaveProperty("article_id"),
```

Comment:
I don't think this is working like you are intending it to. forEach can only take a single callback as an argument. You are passing in 6 of them, so anything after the first callback function is going to be ignored. 

Have a look at the MDN docs for forEach and see if you can refactor to the correct syntax